src/components/Award.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from "react";
+import { AwardPeriod } from "../../generated/definitions/AwardPeriod";
+import { useTranslation } from "react-i18next";
+import { format, parseISO } from "date-fns";
+import viewIcon from "../assets/view.svg";
+import classNames from "classnames";
+import "./Award.css";
+
+type AwardProps = {
+  el: AwardPeriod;
+  index: number;
+  key?: number;
+  popModal: (el: AwardPeriod) => void;
```

Comment:
can you add a comment on what these fields are supposed to be or (better) give them a meaningful name? what's `el` ?